src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.commands.autos;
+
+import java.util.HashMap;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.commands.FollowPathWithEvents;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.Lights;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.drive.DriveCal;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.grabber.Grabber;
+import frc.robot.subsystems.grabber.GrabberCalibrations;
+import frc.robot.commands.autos.components.*;
+
+public class AutoScoreOneFive extends SequentialCommandGroup {
+    private PathPlannerTrajectory oneFive = 
+        PathPlanner.loadPath(
+            "OneFive", 
+            new PathConstraints(
+                DriveCal.MAX_LINEAR_SPEED_METERS_PER_SEC, 
+                DriveCal.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ
+            ));
+
+    private HashMap<String, Command> eventMap = new HashMap<>();
+    /* To-do: put events into event map  */
+    public AutoScoreOneFive(DriveSubsystem drive, Arm arm, Grabber grabber, Lights lights) {
+        addCommands(
+            new AutoScoreOne(false, arm, grabber, lights),
+            new FollowPathWithEvents(drive.followTrajectoryCommand(oneFive, true), oneFive.getMarkers(), eventMap)
+                .withTimeout(2.56),
+            new InstantCommand(() -> grabber.intake()),
+            new WaitUntilCommand(()->grabber.seeGamePiece()),
+            new InstantCommand(() -> grabber.spinMotors(GrabberCalibrations.HOLD_GAME_OBJECT_POWER))
```

Comment:
We have an intake sequence you should be able to use, also you should intake while driving